contracts/snowcones/gauge-proxy-v3.sol

```diff
@@ -0,0 +1,806 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7; //^0.7.5;
+
+//TESTING
+import "hardhat/console.sol";
```

Comment:
Once ready to deploy, I think we can safely remove the hardhat console.

Reply:
Good catch!  Removing this.